kasa/smartcamera/modules/device.py

```diff
@@ -3,7 +3,7 @@
 from __future__ import annotations
 
 from ...feature import Feature
-from ..smartcameramodule import SmartCameraModule
+from ...smartcamera.smartcameramodule import SmartCameraModule
```

Comment:
No need to go that deep higher in the hierarchy, right? `from ..smartcameramodule import SmartCameraModule` should work.

Reply:
Fixed